src/main/java/org/neo4j/junit/jupiter/causal_cluster/DefaultNeo4jServer.java

```diff
@@ -74,20 +83,33 @@
 	}
 
 	@Override
-	public String getDebugLog() {
-		return retrieveLogFile(LogFile.DEBUG);
+	public long getDebugLogPosition() {
+		return retrieveLogFile(LogFile.DEBUG, 0, Stream::count);
+	}
+
+	@Override
+	public String getDebugLogFrom(long offset) {
+		return retrieveLogFile(LogFile.DEBUG, offset, streamToString);
```

Comment:
This is really useful for tests where you want to get the logs *after* you took a particular action. 

It doesn't currently handle log files rolling though - I don't think that anyone should write a test that generates so many logs that the files roll.

Reply:
You never know :D